tests/test_other.py

```diff
@@ -100,6 +100,16 @@ def test_emcc_python_version(self):
       assert expected_call not in output
       assert output == '', output
 
+  def test_emcc_generate_config(self):
+    for compiler in [EMCC, EMXX]:
+      config_path = './emscripten_config'
+      run_process([PYTHON, compiler, '--generate-config', config_path])
+      assert os.path.exists(config_path), 'A config file should have been created at %s' % config_path
+      config_contents = open(config_path).read()
+      self.assertContained('EMSCRIPTEN_ROOT', config_contents)
+      self.assertContained('LLVM_ROOT', config_contents)
+      os.remove(config_path)
```

Comment:
test looks good, thanks!

can remove the last line, the tests are run in a temp dir anyhow, so no manual cleanup is needed

Reply:
Oh, I see. Yeah, it's fine to test just `EMCC` here.